rdr_service/participant_enums.py

```diff
@@ -78,6 +78,7 @@ class QuestionnaireDefinitionStatus(messages.Enum):
 class EnrollmentStatus(messages.Enum):
     """A status reflecting how fully enrolled a participant is"""
 
+    # REGISTERED = 0  # Adding this value breaks metric counts over time.
     INTERESTED = 1
```

Comment:
This is the old version of enrollmentStatus, there is no `REGISTERED` status, I think we can remove this comment line.

Reply:
I want to leave it in as a reminder, when I put it in it broke some metrics dao code that looks up the enum value by array position.

Reply:
I'm so confused about the comment, maybe at least we can change it to something else. 
You add a new status which is not belong to there and break something, that's the expected result. My point is if it is left there, it looks like `REGISTERED` is one of the valid status and we commented it out just because it will break something.  But the truth is `REGISTERED` is not a valid status for the old version of enrollmentStatus at all.

Reply:
Made comment change and commited it.